src/test/daml/Daml/Finance/Instrument/Swap/Test/Asset.daml

```diff
@@ -83,7 +83,7 @@ run = script do
   -- performance payments.
   let
     expectedConsumedQuantities = [qty 0.001675 cashInstrumentCid]
-    expectedProducedQuantities = [qty 0.0801561777 cashInstrumentCid]
+    expectedProducedQuantities = [qty 0.0801561782 cashInstrumentCid]
```

Comment:
Something interested that came out is that by changing the perfomance-linked payoff, we run into rounding issues